source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -13,6 +13,7 @@
 #include <widgets.h>
 
 #include <iostream>
+#include <algorithm>
```

Comment:
Isn't this standard library automatically included through the MaterialX headers?